src/L2GovernorMetadata.sol

```diff
@@ -1,13 +1,12 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
-import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
-import {WormholeReceiver} from "src/WormholeReceiver.sol";
-import {WormholeBase} from "src/WormholeBase.sol";
-
-/// @notice Receives L1 messages with proposal metadata.
-contract L2GovernorMetadata is WormholeReceiver {
-  /// @notice The L1 proposal metadata.
+/// @notice This contract is used by an L2VoteAggregator to store proposal metadata.
+/// It expects to receive proposal metadata from a valid L1 bridge source.
```

Comment:
I think this is a little too specific. An implementation could get proposal metadata via state proofs on L2 or through another protocol like Succint. That being said I think you could remove bridge from this line and modify the line below.

Reply:
thanks, adjusted in 696ed96 -- look ok?